### DISCUSSION TITLE
how to add the tailwindcss  ?

### DISCUSSION THREAD

**Discussion by 729902288** [GetPublii/Publii#1875] [Help]
how to add the tailwindcss  ?

**Comment by michaelfluegge** [comment]
https://tailwindcss.com/docs/installation

**Comment by candidexmedia** [comment]
Are you building a theme from scratch?

**Reply by michaelfluegge** [reply]
@candidexmedia , frankly spoken, I am not sure, whether I want to help someone who asks a question with our any explanation. I mean, at least he or she could at least describe what the goal is, or?
